### PR TITLE
Feature/search/CLK32-903-SearchScreen

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- Ui Screen Home
-Ui Screen Search

### Solution
- update UI Search Screen
- Link sence

### Impact
- Home
- Search

### Screenshots (Optional)
- Dark

- Light

### CODE REVIEW

**Comment by namnh92** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

- [x] I understand that "Search.Placehodel".localized : "General.Search" is now existed in Localized service

- [x] I understand that "Search.Title.Error" is now existed in Localized service

- [x] I understand that "Search.GroupChat" is now existed in Localized service

- [x] I understand that "Search.Message" is now existed in Localized service

- [x] I understand that "Search.People" is now existed in Localized service

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: ClearKeep/Scenes/ContentView.swift [hunk 1]
```diff
@@ -19,14 +19,15 @@ struct ContentView: View {
...
-		}
+//		Group {
+//			if isRunningTests {
+//				Text("Running unit tests")
+//			} else {
+//				LoginView()
+//					.inject(container)
+//			}
+//		}
+		HomeHeaderView(inputStyle: .constant(.default))
```
Comment (namnh92) [inline ClearKeep/Scenes/ContentView.swift]: @Minhdn1 : Revert this code

File: ClearKeep/Scenes/Home/Views/HomeHeaderView.swift [hunk 2]
```diff
@@ -100,39 +126,63 @@ private extension HomeHeaderView {
...
+		HomeHeaderView(inputStyle: .constant(.default))
+	}
+}
+#endif
+
+// MARK: - extension
+#if canImport(UIKit)
+extension View {
+	func hideKeyboard() {
+		UIApplication.shared.sendAction(#selector(UIResponder.resignFirstResponder), to: nil, from: nil, for: nil)
```
Comment (namnh92) [inline ClearKeep/Scenes/Home/Views/HomeHeaderView.swift]: Move to CommonUI Extension View

File: ClearKeep/Scenes/Search/SearchView.swift [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+		colorScheme == .light ? AppTheme.shared.colorSet.background : AppTheme.shared.colorSet.black
+	}
+
+	var forceColorTitle: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.grey3 : AppTheme.shared.colorSet.greyLight
+	}
+	
+	func search() {
+
+	}
```
Comment (namnh92) [inline ClearKeep/Scenes/Search/SearchView.swift]: Remove not use

**Review by namnh92 — CHANGES_REQUESTED** [review]

File: ClearKeep/Scenes/ContentView.swift (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/ContentView.swift]: fix complete

File: ClearKeep/Scenes/Home/Views/HomeHeaderView.swift (hunk 2 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/Home/Views/HomeHeaderView.swift]: fix complete

File: ClearKeep/Scenes/Search/SearchView.swift (hunk 3 above)
Comment (Minhdn1) [reply ClearKeep/Scenes/Search/SearchView.swift]: fix complete

**Review by namnh92 — APPROVED** [review]

**Pull request merged**